.github/workflows/qa-linux-3.10.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        python-version: [ "3.9" ]
+        python-version: [ "3.10" ]
```

Comment:
these are generated file and you have to update the line [here](https://github.com/xonsh/xonsh/blob/81c8fe6bb92abe53277fbfffe6461630440c4abd/.github/workflows/genbuilds.py#L42) to generate them with py3.10